integration/metadata_globals_queries_test.go

```diff
@@ -344,6 +344,16 @@ CREATEEXTTABLE (protocol='gphdfs', type='writable')`
 
 			for _, role := range results {
 				if role.Name == "role1" {
+					if connectionPool.Version.IsCBDB() {
+						// Cloudberry uses SCRAM-SHA-256 by default, while the test was
+						// written expecting an MD5 hash. The exact SCRAM hash is
+						// unpredictable due to salting, so we only check its prefix.
+						Expect(role.Password).To(HavePrefix("SCRAM-SHA-256$"))
```

Comment:
Is there a way to get the salt value and compute the hash value?

Reply:
Thank you for the thoughtful review and the suggestion to extract the salt value and compute the hash. This is technically feasible and shows good attention to detail.

While it's possible to extract the salt and recompute the SCRAM hash, I believe this exceeds the responsibility scope of this integration test. Our test goal is to verify that the `GetRoles()` function correctly extracts and maps role information from the database, not to verify PostgreSQL's SCRAM algorithm implementation.

The current format validation is sufficient to ensure:
1. Data is correctly extracted  
2. SCRAM format is properly recognized

Re-implementing the SCRAM algorithm would add unnecessary complexity and maintenance burden with limited benefit.

Reply:
Looks good.